### DISCUSSION TITLE
Kibana enhanced table plugin error

### DISCUSSION THREAD

**Discussion by Rushikesh210** [fbaligand/kibana-enhanced-table#333] [Q&A] [answered]
We have setup kibana development environment locally and installed enhanced-table plugin using plugin_install.js script by providing zip file of the plugin, it gets installed successfully.
Then I want to create one table using this enhanced table plugin, I am following below steps:-
1. In metrics if i choose aggregation then its shows error "Cannot read properties of undefined (reading 'call')-
 ![image](
![screenshot2](https://github.com/fbaligand/kibana-enhanced-table/assets/90234168/2d9cf713-bea4-416c-a468-278a511c97df) 
After that if i choose any field in metrics or buckets its shows below error " i is not a function "-
![image](
![screenshot1](https://github.com/fbaligand/kibana-enhanced-table/assets/90234168/1c87596c-a648-4601-8b73-08f16cda4aae)
please help me for that.

**Comment by fbaligand** [comment]
kibana dev mode is done for developing a kibana plugin. 
 so if you only wish to “run” the plugin, I invite you to run kibana in prod mode, and use the kibana package provided on elastic.co/downloads
 
If you want to make some changes in enhanced table plugin code, you must not install zip package. I invite you to follow indications here:
https://github.com/fbaligand/kibana-enhanced-table/#development

If you still have errors, indicate me which kibana version you use.

**Reply by Rushikesh210** [reply, marked as answer]
Thank you . I will try to run kibana enhanced table plugin in development mode .
